### PR TITLE
iOS 13 Dark mode support in AlertView 

### PR DESCRIPTION
**Pull request by ftp27**
There is a small fix to support dark background in AlertView and few changes in Example storyboard to make it look nicer in iOS 13.

### CODE REVIEW

**Comment by jaketesler** [comment]
Is it possible for someone to merge this PR + issue a new release on GH with the fix?

**Comment by andre719mv** [comment]
Also it will be good to fix background and border for textfield in dark mode.

**Comment by jaketesler** [comment]
Any updates on this PR? :)

**Comment by ftp27** [comment]
@andre719mv Now it works, I guess )

**Comment by jaketesler** [comment]
Can someone merge this + release an update? Our app could really use dark mode alert support 💯

**Comment by sberrevoets** [comment]
Sorry, time is in short supply for me nowadays. I'll do my best to get to this soon but until then I recommend using this branch.

File: Example/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,12 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13189.4" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="uvy-DI-gcx">
-    <device id="retina4_0" orientation="portrait">
-        <adaptation id="fullscreen"/>
-    </device>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14868" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="uvy-DI-gcx">
+    <device id="retina4_0" orientation="portrait" appearance="light"/>
```
Comment (sberrevoets) [inline Example/Base.lproj/Main.storyboard]: There are a bunch of constraint/frame changes in here, can you remove those as I don't think they're relevant to this PR

File: Source/Text Fields/TextFieldCell.swift [hunk 2]
```diff
@@ -18,11 +18,8 @@ final class TextFieldCell: UITableViewCell {
         didSet {
             self.textField?.font = self.visualStyle?.textFieldFont
             self.borderView.backgroundColor = self.visualStyle?.textFieldBorderColor
-
-            guard let padding = self.visualStyle?.textFieldMargins else {
-                return
-            }
-
+            self.textFieldContainer.backgroundColor = self.visualStyle?.textFieldBackgroundColor
+            guard let padding = self.visualStyle?.textFieldMargins else { return }
```
Comment (sberrevoets) [inline Source/Text Fields/TextFieldCell.swift]: Can you re-format this `guard` to the previous version?

**Comment by ftp27** [comment]
@sberrevoets Is it ok now?

**Comment by sberrevoets** [comment]
Thanks, version 11 is now available through CocoaPods

**Pull request merged**